playfab-docs/features/pricing/development-mode.md

```diff
@@ -75,7 +75,7 @@ There are some studio and title limits associated with development mode:
 | Insights Credits | Insights Performance Level 1 |
 
 **Definition of Unique Users:**
-Unique Users are simply the total number of player profiles you have in your title, which is different from MAU - you can delete player profiles through the Players page in Game Manager to reduce the total number of Unique Users if you're approaching the limit but aren't ready to launch your title
+Unique Users are the total number of players created in your title which is distinct from monthly active users or daily active users.
```

Comment:
Two nit picks:
- do we still want MAU DAU as acronyms?
- Should this have a period at the end? (it didn't before, not sure if that's a consistent pattern)

Reply:
MAU and DAU are acronyms that are common in the gaming industry and typically understood by our customers. They are also frequently used in the billing models by our competitors which is why it's important to have the distinction because many customers expect DAU and MAU to be billing metrics.

This is what it looks like today

<img width="784" alt="image" src="https://github.com/MicrosoftDocs/playfab-docs/assets/18044509/acca56fa-f7c4-4dc6-a073-bc6ec907a0f4">